templates/named.conf.options.erb

```diff
@@ -153,4 +153,24 @@ logging {
         category default { syslog_errors;  };
         category lame-servers { null; };
 };
+<%- else -%>
```

Comment:
It is a `else` with an `if` inside. Maybe I should write it with an `elsif`.
The idea is that if you enable `query_log_enable`, then you get the default configuration, but if you don't *and* you configure `log_categories` and `log_channels` you could set a more fine logging configuration